src/software/backend/radio/mrf/messages.h

```diff
@@ -104,10 +104,10 @@ namespace MRF
         "wheel 2 motor hot",
         "wheel 3 motor hot",
         "dribbler motor hot",
-        "wheel 0 encoder not commutating",
-        "wheel 1 encoder not commutating",
-        "wheel 2 encoder not commutating",
-        "wheel 3 encoder not commutating",
+        "wheel 0 encoder not commuting",
+        "wheel 1 encoder not commuting",
+        "wheel 2 encoder not commuting",
+        "wheel 3 encoder not commuting",
```

Comment:
What does `commuting` stand for in this context?

Reply:
This was a formatting error. It has since been reverted.